cluster/terraform_kubernetes/alertmanager.tf

```diff
@@ -0,0 +1,165 @@
+data "azurerm_key_vault" "key_vault_slack" {
+  name                = var.cluster_kv
+  resource_group_name = var.resource_group_name
+}
+data "azurerm_key_vault_secret" "slack_secret" {
+  key_vault_id = data.azurerm_key_vault.key_vault_slack.id
```

Comment:
should be able to use data.azurerm_key_vault.key_vault.id

Reply:
Fixed